### DISCUSSION TITLE
Which model is qwen-max in the first picture in the second picture?

### DISCUSSION THREAD

**Discussion by dosnshcu** [langgenius/dify#10268] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Which model is qwen-max in the first picture in the second picture?

### 2. Additional context or comments

I need to request a model concurrency limit with the model vendor, but I'm not sure which model this refers to
![image](https://github.com/user-attachments/assets/40455fef-59b7-45cc-a458-ad33cc009fe6)
![image](https://github.com/user-attachments/assets/a88256c6-0f5c-4cfa-9f54-f7596fa3ace7)

**Comment by crazywoola** [comment]
This is the latest one

https://github.com/langgenius/dify/blob/ecc951609d4988424bfba58094d6e256ae86144e/api/core/model_runtime/model_providers/tongyi/llm/qwen-max-latest.yaml#L3

This is the qwen-max

https://github.com/langgenius/dify/blob/9b32bfb3dbdac71f214e612c7b5482549aadf3d9/api/core/model_runtime/model_providers/tongyi/llm/qwen-max.yaml#L3

**Reply by dosnshcu** [reply]
But I don't have qwen-max-latest inside my ali model and there is no import button, how should I import qwen-max-latest
![image](https://github.com/user-attachments/assets/6d283ba8-f39f-46a5-9b20-8adb9b6c399d)
